qutebrowser/browser/commands.py

```diff
@@ -976,6 +978,9 @@ def tab_focus(self, index: typing.Union[str, int]=None, count=None):
         if index is None:
             self.tab_next()
             return
+        if index == 0:
+            self._tab_focus_last()
+            return
```

Comment:
The idea was to focus the rightmost tab with `0T`, not the last focused one - the docstring above should then also be adjusted accordingly.